backend/curriculum_tracking/models.py

```diff
@@ -1383,3 +1384,20 @@ def add_collaborator(self, user, add_as_project_reviewer):
                 if self.recruit_project:
                     self.recruit_project.reviewer_users.add(user)
         self.save()
+
+    def get_users_that_reviewed_since_last_review_request(self):
+        if self.review_request_time is None:
+            return []
+
+        if self.content_item.content_type == ContentItem.PROJECT:
+            ReviewClass = RecruitProjectReview
+
+        elif self.content_item.content_type == ContentItem.TOPIC:
+            ReviewClass = TopicReview
+
+        reviews = ReviewClass.objects.filter(
+            timestamp__gte=self.review_request_time
```

Comment:
Oops, forgot something here. 

We need to make sure that we are filtering by the TopicProgress or RecruitProject entity as well. Not just timestamp.

Eg: If I have 2 projects sucht that project 1 has a bunch of reviews and project 2 des not, then `project2.get_users_that_reviewed_since_last_review_request()` shouldn't return anything user ids.

So basically there is a filter missing. 

The streucture of the code can be the same, except you can do this sort of thing:

```
        if self.content_item.content_type == ContentItem.PROJECT:
            # ReviewClass = RecruitProjectReview not this 
           reviews = RecruitProjectReview.objects.filter(recruit_project = self)

      ....
   # then we chain filters instead of defining the query from scratch
    reviews = reviews.filter(timestamp__gte=self.review_request_time)

```